### ISSUE TITLE
New PhpStorm crash log after upgrading to 0.12.3

### ISSUE THREAD

**Issue by cirdog** [Haehnchen/idea-php-laravel-plugin#99]
I've just upgraded to 0.12.3 after waiting it out (was crashing on my other comptuer), then after restart I encountered this crash and captured the log. Thanks for looking into it!
```
Read access is allowed from event dispatch thread or inside read-action only (see com.intellij.openapi.application.Application.runReadAction())
Details: Current thread: Thread[ApplicationImpl pooled thread 13,4,Idea Thread Group] 1012090208
; dispatch thread: false; isDispatchThread(): false
SystemEventQueueThread: Thread[AWT-EventQueue-0 2016.3.2#PS-163.10504.2 PhpStorm, eap:false, os:Windows 10 10.0, java-version:Oracle Corporation 1.8.0_91-b15,6,main] 272495274
java.lang.Throwable
	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:150)
	at com.intellij.openapi.application.impl.ApplicationImpl.assertReadAccessAllowed(ApplicationImpl.java:979)
	at com.intellij.util.indexing.FileBasedIndexImpl.ensureUpToDate(FileBasedIndexImpl.java:664)
	at com.intellij.util.indexing.FileBasedIndexImpl.a(FileBasedIndexImpl.java:839)
	at com.intellij.util.indexing.FileBasedIndexImpl.a(FileBasedIndexImpl.java:921)
	at com.intellij.util.indexing.FileBasedIndexImpl.a(FileBasedIndexImpl.java:895)
	at com.intellij.util.indexing.FileBasedIndexImpl.processValues(FileBasedIndexImpl.java:783)
	at com.intellij.psi.search.FilenameIndex.processFilesByName(FilenameIndex.java:127)
	at com.intellij.psi.search.FilenameIndex.processFilesByName(FilenameIndex.java:113)
	at com.intellij.psi.search.FilenameIndex.getFilesByName(FilenameIndex.java:189)
	at com.intellij.psi.search.FilenameIndex.getFilesByName(FilenameIndex.java:104)
	at de.espend.idea.laravel.view.ViewCollector.collectIdeJsonBladePaths(ViewCollector.java:62)
	at de.espend.idea.laravel.view.ViewCollector.getPaths(ViewCollector.java:47)
	at de.espend.idea.laravel.ui.TemplateSettingsForm.attachItems(TemplateSettingsForm.java:70)
	at de.espend.idea.laravel.ui.TemplateSettingsForm.<init>(TemplateSettingsForm.java:49)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.picocontainer.defaults.InstantiatingComponentAdapter.newInstance(InstantiatingComponentAdapter.java:193)
	at org.picocontainer.defaults.ConstructorInjectionComponentAdapter$1.run(ConstructorInjectionComponentAdapter.java:220)
	at org.picocontainer.defaults.ThreadLocalCyclicDependencyGuard.observe(ThreadLocalCyclicDependencyGuard.java:53)
	at org.picocontainer.defaults.ConstructorInjectionComponentAdapter.getComponentInstance(ConstructorInjectionComponentAdapter.java:248)
	at com.intellij.util.pico.CachingConstructorInjectionComponentAdapter.getComponentInstance(CachingConstructorInjectionComponentAdapter.java:58)
	at com.intellij.openapi.extensions.AbstractExtensionPointBean.instantiate(AbstractExtensionPointBean.java:75)
	at com.intellij.openapi.options.ConfigurableEP$ClassProducer.createElement(ConfigurableEP.java:347)
	at com.intellij.openapi.options.ConfigurableEP.createConfigurable(ConfigurableEP.java:273)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.a(ConfigurableWrapper.java:53)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.getConfigurable(ConfigurableWrapper.java:112)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.cast(ConfigurableWrapper.java:93)
	at com.intellij.openapi.options.ex.ConfigurableCardPanel.prepare(ConfigurableCardPanel.java:42)
	at com.intellij.openapi.options.ex.ConfigurableCardPanel.prepare(ConfigurableCardPanel.java:35)
	at com.intellij.ui.CardLayoutPanel.a(CardLayoutPanel.java:128)
	at com.intellij.openapi.application.impl.ApplicationImpl$2.run(ApplicationImpl.java:309)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by naveensnayak** [comment]
+1

**Comment by bluegeek** [comment]
# +1.I got this too

**Comment by Haehnchen** [comment]
sry guys, but now fixed with `0.12.4`

**Comment by cirdog** [comment]
Thanks!! Trying out 0.12.4 now.
